app/ports/models/stats.py

```diff
@@ -38,11 +40,16 @@ def populate(cls, json_object, timestamp):
         uuid_obj, created = UUID.objects.get_or_create(uuid=json_object['id'])
         sub = Submission()
         sub.user = uuid_obj
-        sub.os_version = json_object['os'].get('osx_version')
+        os_version = json_object['os'].get('osx_version')
+        cxx_stdlib = json_object['os'].get('cxx_stdlib')
+        if cxx_stdlib is None and LooseVersion(os_version) >= LooseVersion('10.9'):
```

Comment:
What if `os_version` is `None` / what if this is not macOS?

We probably want a test to make sure it behaves correctly.